js/ui/drop_down_button.js

```diff
@@ -174,7 +174,8 @@ let DropDownButton = Widget.inherit({
         this.$element().addClass(DROP_DOWN_BUTTON_CLASS);
         this._renderButtonGroup();
         this._loadSelectedItem().done((selectedItem) => {
-            this.option("selectedItem", selectedItem);
+            this.option().selectedItem = selectedItem;
```

Comment:
You use it to avoid an `optionChange`? I think better use more clear way. E.g. add method like `setOptionSilent`